### PR TITLE
feat(i18n): add formatter support of the russian language

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [x] New feature, 增加对 Russian 的 formatter 补充，ref #2853

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/utils/formatter.ts [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import { isNaN, toNumber } from 'lodash';
 import { PRECISION } from '../common/constant';
-import { getLang } from '../common/i18n';
+import { getLang, type LangType } from '../common/i18n';
 
-const FORMATTERS = {
+const FORMATTERS: { [K in LangType]: [string[], number[]] } = {
   en_US: ['KMBTP'.split(''), [1e3, 1e3, 1e3, 1e3, 1e3]],
+  ru: [
```
Comment (lijinke666) [inline packages/s2-core/src/utils/formatter.ts:7]: 这个 lang 是不是应该叫 `ru_RU`

Comment (wjgogogo) [reply packages/s2-core/src/utils/formatter.ts:7]: 之前那个  PR 里面叫 ru, 我改一下吧

**Review by lijinke666 — APPROVED** [review]

**Comment by lijinke666** [comment]
咋 commit 又用英文了

**Comment by wjgogogo** [comment]
> 咋 commit 又用英文了

借用了下之前的 PR 的格式

**Pull request merged**